pvlib/modelchain.py

```diff
@@ -276,14 +280,29 @@ class ModelChainResult:
     tracking: Optional[pd.DataFrame] = field(default=None)
```

Comment:
`tracking` is not `PerArray`, so maybe it should be above the comment (at least for now)